velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -172,7 +212,13 @@ bool ExpressionVerifier::verify(
   try {
     exec::ExprSet exprSetCommon(
         {plan}, execCtx_, !options_.disableConstantFolding);
-    exec::EvalCtx evalCtxCommon(execCtx_, &exprSetCommon, rowVector.get());
+    auto wrappedRowVector = wrapColumnsInLazy(rowVector, columnsToWarpInLazy);
```

Comment:
maybe encapsulate the debuging logic in the wrapColumnsInLazy() function so that you could keep this part simpler:

```
exec::ExprSet exprSetCommon(
        {plan}, execCtx_, !options_.disableConstantFolding);
exec::EvalCtx evalCtxCommon(
        execCtx_, &exprSetCommon, wrapColumnsInLazy(rowVector));
```